website/docs/migrating-from-docker/managing-docker-compatibility.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 5
+title: Managing Docker compatibility
+description: Covers configurations available to manage docker compatibility
+keywords: [podman desktop, podman, docker, compatibility]
+tags: [managing-docker-compatibility]
+---
+
+# Managing Docker compatibility
+
+With Podman Desktop, you can configure a Docker-compatible environment to run your Docker applications on a Podman engine, rather than on a Docker engine.
+
+As a developer, you can:
+
+- Check the socket mapping status along with socket details, such as client name, Docker version, and OS/Arch.
+- Use all Docker commands with Podman. For example, you can run the `docker run` command on the Podman engine to start a container.
+- Use Docker Compose commands with Podman by installing and setting up the [Compose extension](/docs/compose). For example, you can run the `docker compose up` command on the Podman engine to start your Docker application. You can place your Docker Compose file in a _/var/compose/_ directory so that Podman can access it. If the Compose CLI is not installed, you get the install option in the **Docker Compatibility** settings.
```

Comment:
if you install the compose extension, then you have `podman compose` working 

`to start your Docker application` => compose application

also it's compose v2 application (not v1)

> You can place your Docker Compose file in a _/var/compose/_ directory 
I would expect something more like 'in a working directory'. Usually you work within your home folder not in /var folder (also it does not exist on Windows)